Vision/Vision.bs

```diff
@@ -93,9 +93,9 @@ Markup Shorthands: markdown yes
 
 # W3C's Vision for the World Wide Web # {#vision-web}
 
-	* The Web is for <a href="https://www.w3.org/TR/ethical-web-principles/#allpeople">all people</a>.
-	* The Web is designed for the <a href="https://www.w3.org/TR/ethical-web-principles/#noharm">good of humanity</a>.
-	* The Web must be <a href="https://www.w3.org/TR/ethical-web-principles/#privacy">secure and respect people's privacy</a>.
+	* The Web is for <a href="https://www.w3.org/TR/ethical-web-principles/#allpeople">all humanity</a>.
```

Comment:
```suggestion
	* The Web is for <a href="https://www.w3.org/TR/ethical-web-principles/#allpeople">all people</a>.
```

Reply:
Ok, given:
- "All humanity" is the phrasing that was here originally (prior to recent edits attempting to fix #211), and had the consensus of the AB and the VisionTF at least in the past.
- No one, anywhere that I can see, not even in #211, was complaining about it.
- You yourself admitted in Slack that you find rephrasing with "all people" to be less eloquent than the original "all humanity".

... what's your motivation for proposing the change?

(It's not that I have a particular objection to this change, but I'm not seeing the benefit.)